### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by duxv**

### CODE REVIEW

File: py/bigint.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func BigIntCheckExact(obj Object) (*BigInt, error) {
 	return bigInt, nil
 }
 
-// Checks that obj is exactly a bigInd and returns an error if not
+// Checks that obj is exactly a bigInt and returns an error if not
```
Comment (sbinet) [inline py/bigint.go:58]:
```suggestion
// Checks that obj is exactly a BigInt and returns an error if not
```

**Comment by duxv** [comment]
> Hi @duxv, your interest is appreciated and I've added that typo fix in a PR in the coming weeks.
> 
> If you are serious about contributing toward the project, I am willing guide you and make suggestions if that sounds interesting. Or lmk how you might be interested in supporting gpython or what you might be interested in helping with.

Hi. Thanks for the offer, but I don't believe that my knowledge is far enough to support such a project as gpython. So far, I've been using it only to further my knowledge by reading the source code and trying to understand it. I wanna thank you, the creators of gpython, for creating such an awesome project that taught me so much.

**Comment by sbinet** [comment]
hi,
gpython accepts all contributions.
fixing typos included :)
(everybody, on any project, started with small-ish PRs to whet their appetite)

**Comment by sbinet** [comment]
ping?

**Comment by sbinet** [comment]
_pong?_

**Comment by sbinet** [comment]
timeout.

**Pull request closed without merging**